skimage/feature/_texture.pyx

```diff
@@ -148,14 +141,15 @@ def _local_binary_pattern(double[:, ::1] image,
 
     # To compute the variance features
     cdef double sum_, var_, texture_i
+    cdef cnp.float64_t zero = 0
 
     with nogil:
         for r in range(image.shape[0]):
             for c in range(image.shape[1]):
                 for i in range(P):
-                    texture[i] = bilinear_interpolation(&image[0, 0], rows, cols,
-                                                        r + rp[i], c + cp[i],
-                                                        'C', 0)
+                    bilinear_interpolation[cnp.float64_t, double, double](
```

Comment:
Whoa, what's going on here? Do you have a reference for this syntax?

Reply:
https://cython.readthedocs.io/en/latest/src/userguide/fusedtypes.html#indexing

fused type indexing. I didn't want to make changes to this function.